### DISCUSSION TITLE
Several key-value stores?

### DISCUSSION THREAD

**Discussion by PopeAndrewJ** [koculu/ZoneTree#52] [Q&A]
Hello, 

Thank you for this really cool project! I'm looking for a kv store for c#, and zone tree looks like a great fit.  I admit not having much experience with kv stores, so maybe my question will show that....

Anyway, how can I store differently typed values in separate trees? I didn't see any examples of storing more than one type of value, so I tried it.  I made a <int,string> tree and a <int,int> tree, but it throws an exception complaining about the delete options not being setup. I must have some incorrect idea of how it should work....

Thanks again

**Comment by koculu** [comment]
Hi @PopeAndrewJ 
The section deletion-markers-and-ttl describes the configuration for deleting the keys.

https://zonetree.dev/docs/concepts/deletion-markers-and-ttl/

Getting Started: https://zonetree.dev/docs/getting-started/

Thank you.
